scripts/ci/build-and-test.sh

```diff
@@ -14,42 +14,12 @@ start_tunnel
 
 wait_for_tunnel
 if is_lint; then
-  npm run tslint  
-  npm run ci:forbidden-identifiers
-  npm run stylelint
-elif is_circular_deps_check; then
-  # Build first because madge needs the JavaScript output.
-  ng build
-  npm run check-circular-deps
+  $(npm bin)/gulp ci:lint
```

Comment:
Not sure if this would work, but could you add at the beginning `alias gulp='$(npm bin)/gulp'` and then just use `gulp` afterwards?